README.md

```diff
@@ -168,6 +168,38 @@ jobs:
             npm run build
 ```
 
+## Releasing monorepo packages
+
+For use in a monorepo this action expects two new inputs `monorepo-package` and `monorepo-root`, they will be used to run command and publish from the package folder instead of the project root.
```

Comment:
what is the "project root" here? I don't see this as a configurable option, so at the moment I assume this means "repository root"?

Reply:
if you had a single package, but it lived in a subdirectory of your repository, could monorepo-root be used without monorepo-package?

Perhaps just a "root" is needed, or maybe "working-directory", "workingDir"?

Reply:
Given a repository of the structure

```
yarn.lock
|-packages
  |-my-package-1
  |-my-package-2
```

I would expect the name "monorepo-root" to mean where your lock file is.

But I think you are using it here to refer to the packages directory.

I wonder if packages could include paths, so for the above you would have

```yml
root: .
packages:
  - packages/my-package-1
  - packages/my-package-2
```

And though possibly uncommon would support a repository that looked like:


``
my-java-project
|-Jenkinsfile.jank
my-node-project
|-yarn.lock
  |-packages
    |-my-package-1
    |-my-package-2
```

with

```yml
root: my-node-project
packages:
  - packages/my-package-1
  - packages/my-package-2
```

Reply:
> if you had a single package, but it lived in a subdirectory of your repository, could monorepo-root be used without monorepo-package?
> 
> Perhaps just a "root" is needed, or maybe "working-directory", "workingDir"?

I agree it could be useful but I think it would be better suited for another PR since it isn't related to monorepos